transactions/src/main/java/org/ehcache/transactions/xa/internal/xml/TxCacheManagerServiceConfigurationParser.java

```diff
@@ -71,11 +83,42 @@ public URI getNamespace() {
 
   @Override
   public Class<TransactionManagerProvider> getServiceType() {
-    return null;
+    return TransactionManagerProvider.class;
   }
 
   @Override
   public Element unparseServiceCreationConfiguration(ServiceCreationConfiguration<TransactionManagerProvider> serviceCreationConfiguration) {
-    throw new UnsupportedOperationException("Not yet implemented");
+    try {
+      validateParameterForTranslationToServiceConfig(serviceCreationConfiguration);
+      Document doc = createDocumentRoot();
+      LookupTransactionManagerProviderConfiguration lookupTransactionManagerProviderConfiguration = (LookupTransactionManagerProviderConfiguration) serviceCreationConfiguration;
+      Element rootElement = createRootElement(doc, lookupTransactionManagerProviderConfiguration);
+      return rootElement;
+    } catch (SAXException | ParserConfigurationException | IOException e) {
+      throw new XmlConfigurationException(e);
+    }
+  }
+
+  private void validateParameterForTranslationToServiceConfig(ServiceCreationConfiguration<TransactionManagerProvider> serviceCreationConfiguration) {
+    Objects.requireNonNull(serviceCreationConfiguration, "ServiceCreationConfiguration must not be NULL");
+    if (!(serviceCreationConfiguration instanceof LookupTransactionManagerProviderConfiguration)) {
+      throw new IllegalArgumentException("Parameter serviceCreationConfiguration must be of type LookupTransactionManagerProviderConfiguration."
+                                         + "Provided type of parameter is : " + serviceCreationConfiguration.getClass());
+    }
   }
+
+  private Element createRootElement(Document doc, LookupTransactionManagerProviderConfiguration lookupTransactionManagerProviderConfiguration) {
+    Element rootElement = doc.createElement(TRANSACTION_NAMESPACE_PREFIX + COLON + TRANSACTION_ELEMENT_NAME);
+    rootElement.setAttributeNS(XMLConstants.XMLNS_ATTRIBUTE_NS_URI, "xmlns:" + TRANSACTION_NAMESPACE_PREFIX, NAMESPACE.toString());
+    rootElement.setAttribute(TRANSACTION_LOOKUP_CLASS, lookupTransactionManagerProviderConfiguration.getTransactionManagerLookup()
+      .getName());
+    return rootElement;
+  }
+
+  private Document createDocumentRoot() throws IOException, SAXException, ParserConfigurationException {
```

Comment:
Shouldn't this document be correctly namespaced otherwise you'll run in to issues with jaxb further down the line since it will have no way of validating the elements membership in the correct substitution group.